docs/environments.md

```
@@ -49,16 +49,22 @@ bootstrapped and the GitHub Environment holds a `KUBECONFIG`.
-| `test2` | `test2.eduide.student.k8s.aet.cit.tum.de` |
-| `test3` | `test3.eduide.student.k8s.aet.cit.tum.de` |
-| `e2e-test` | `e2e.eduide.student.k8s.aet.cit.tum.de` |
-| `staging` | `staging.eduide.student.k8s.aet.cit.tum.de` |
-| `tum-production` | `eduide.artemis.cit.tum.de` - see below |
-| `bonn` | `bonn.eduide.aet.cit.tum.de` |
-| `mannheim` | `mannheim.eduide.aet.cit.tum.de` |
+An environment **is** its landing hostname. The directory under
+`environments/`, the GitHub Environment that holds its secrets, and the host it
+serves are one string:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)